video-object-detection/cfg/coco.data

```diff
@@ -0,0 +1,8 @@
+classes= 80
+train  = /home/pjreddie/data/coco/trainvalno5k.txt
```

Comment:
Are these two references to `/home/pjreddie` correct?

Reply:
They're not correct. He's the author of Darknet. Will remove them as we don't need them for inference.
FWIW the issue comes from the deep learning server example.